src/components/DatePicker/DatePicker.vue

```diff
@@ -163,4 +163,16 @@ export default class DatePicker extends Vue {
 ::v-deep .v-icon.v-icon.v-icon--link {
   cursor: text;
 }
+
+::v-deep .theme--light.v-icon.v-icon.v-icon--disabled {
```

Comment:
Do these need to be v-deep, since you're not reaching in to a rendered component now?

Reply:
PS I doubt that `.theme--light` is useful here and in selectors below.

Reply:
Yes, we do need v-deep because it is inside v-text-field.

![image](https://user-images.githubusercontent.com/78114138/117089225-b650ed80-ad09-11eb-8059-3aaa845292d3.png)
Maybe for the other I can get rid of .theme--light but for line 167 if it doesn't match exactly the selector, it doesn't override it.

Reply:
I fixed one css selector but the other 2 need the `.theme--light`